COMPILING.md

```diff
@@ -66,7 +66,7 @@ make
 sudo make install # optional
 ```
 
-To control the server with systemd, see this [unit file example](https://github.com/jamulussoftware/jamulus/blob/master/distributions/jamulus-server.service). See also runtime (https://jamulus.io/wiki/Command-Line-Options), and (https://jamulus.io/wiki/Tips-Tricks-More#controlling-recording-on-linux-headless-servers) on controlling recordings on headless servers.
+To control the server with systemd, see this [unit file example](https://github.com/jamulussoftware/jamulus/blob/master/distributions/jamulus-server.service). See also runtime [configuration options](https://jamulus.io/wiki/Command-Line-Options), and [this information](https://jamulus.io/wiki/Tips-Tricks-More#controlling-recording-on-linux-headless-servers) on controlling recordings on headless servers.
```

Comment:
In the anchor link: `#controlling-recording-on-linux-headless-servers` you need to add an 's' to recording -> recording**s**. This came up with the same link in the website documentation.

Reply:
done in bf10c1b